proposals/0282-record-dot-syntax.rst

```diff
@@ -212,7 +212,7 @@ Symbol Occurence
 
 [Field]
 :raw-html:`<br />`
-     *field*   →   *varid*
+     *field*   →   *varid* | *reservedid*
```

Comment:
I would be content with a single change:

```suggestion
     *field*   →   *varid* | *reservedid*
     
(We include *reservedid* so that `foo.type` is allowed. This does not affect what identifiers are allowed as record fields, where *reservedid*s such as `type` are not allowed.) 
```